addons/account_payment/views/account_portal_templates.xml

```diff
@@ -45,7 +45,7 @@
                     <div class="modal-content">
                         <div class="modal-header">
                             <h3 class="modal-title">Pay with</h3>
-                            <button type="button" class="close" data-dismiss="modal">&amp;times;</button>
+                            <button type="button" class="close" data-dismiss="modal" t-translation="off">×</button>
```

Comment:
1 character translaions should [be ignored](https://github.com/odoo/odoo/blob/0e2c55eec07600585121acf560ec8ffebde442e3/odoo/tools/translate.py#L168) from the translation parsing `t-translation` should not be needed here